Gruntfile.js

```diff
@@ -183,7 +183,7 @@ module.exports = exports = function(grunt) {
                     'dist/localforage.nopromises.min.js': [
                         'dist/localforage.nopromises.js'
                     ],
-                    'site/localforage.min.js': ['dist/localforage.js']
+                    'site/javascripts/localforage.min.js': ['dist/localforage.js']
```

Comment:
Oh, can we copy this or reference it from the root level? I'm not sure if many people use it, but the point in generating that file was so that people could hotlink to it (and the site itself links to it in the top of the site), so I'd rather not change the URL :smile: